### PR TITLE
Add 'compose' folder and sublime text files to gitignore

### PR DESCRIPTION
**Pull request by m-ober**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,6 +9,7 @@
 target
 /core
 /bd
```
Comment (sommerlukas) [inline .gitignore]: You should remove the `bd`-folder, because the name has been changed to `compose`.

**Comment by m-ober** [comment]
Thanks for your feedback – if old entries should be removed, I guess the following can also be removed?  They are no longer in the repository:

```
boot/pynq
boot/zc706
boot/zedboard
boot/rootfs.img
boot/uenv/uEnv-pynq.txt
bin/itapasco
```

**Comment by sommerlukas** [comment]
Yes, I guess all these can be removed, too. Overall, it looks as the `.gitignore` does not match the new folder-structure currently implemented on `develop` at all. 
Can you update the whole `.gitignore` as part of this pull-request? Otherwise, just open an issue and we will take care of it.

**Comment by m-ober** [comment]
Updated it to the best of my knowledge. You can merge it or close the PR and use the changes as base for a (more) complete update of the gitignore, if you like.

**Pull request closed without merging**